network-api/networkapi/wagtailpages/pagemodels/campaigns.py

```diff
@@ -171,7 +171,6 @@ class Petition(TranslatableMixin, CTA):
         max_length=20,
         help_text="Which campaign identifier should this petition be tied to?",
```

Comment:
Since we are updating this field, could we also update the help text to have "Salesforce" for extra clarity? e.g., something like `"Which Salesforce campaign identifier should this petition be tied to?"`
```suggestion
        help_text="Which Salesforce campaign identifier should this petition be tied to?",
```

Reply:
Thanks @mmmavis! This is a great suggestion and has been implemented now.